docs/meetings.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Replace with `#  Weekly Community Meetings`

Reply:
I thought doc purely technical. I prefer meeting.md can be merged into the top level readme.